dev/include_dune

```diff
@@ -0,0 +1,85 @@
+
+(* File to include to install the pretty-printers in the ocaml toplevel *)
+
+(* Typical usage :
+
+   $ coqtop.byte    # or even better : rlwrap coqtop.byte
+   Coq < Drop.
+   # #use "include";;
+
+   Alternatively, you can avoid typing #use "include" after each Drop
+   by adding the following lines in your $HOME/.ocamlinit :
+
+   #directory "+compiler-libs";;
+   if Filename.basename Sys.argv.(0) = "coqtop.byte"
+   then ignore (Toploop.use_silently Format.std_formatter "dev/include")
+*)
+
+(* For OCaml 3.10.x:
```

Comment:
Seems like this was copied from somewhere else. You can probably delete this comment.

Reply:
I will indeed refactor a bit more.